Jenkinsfile

```diff
@@ -56,6 +56,13 @@ pipeline {
               '''
             }
           }
+          post {
+            failure {
```

Comment:
I think you probably want regression, not failure here.

Reply:
done, though I think we actually want `failure` because I want it to be annoying if it keeps failing. But we can change it later.

Reply:
Alright. Note that failure will not run if the build is unstable; not sure if you ever get unstable builds, but something to consider.

Reply:
Changed back to `failure`, we can change to `regression` if it becomes annoying.